conda/win/meta.yaml

```diff
@@ -21,6 +21,8 @@ requirements:
 build:
   string: py{{ environ['PY_VER'].replace('.', '') }}_{{ 'debug' if environ['CMAKE_BUILD_TYPE'] == 'Debug' else '0' }}
   script:
+    - copy /y {{INSTALL_DIR}}\bin\psi4     {{PREFIX}}\Scripts
+    - copy /y {{INSTALL_DIR}}\bin\psi4.bat {{PREFIX}}\Scripts
```

Comment:
Is this patching a failure at the CMake level?

Reply:
I would say, this is more a peculiarity of Conda on Windows.